src/nodes/agent/handlers/VaultsGitInfoGet.ts

```diff
@@ -3,7 +3,7 @@ import type Logger from '@matrixai/logger';
 import type { ContextTimed } from '@matrixai/contexts';
 import type ACL from '../../../acl/ACL';
 import type VaultManager from '../../../vaults/VaultManager';
-import type { JSONRPCRequest } from '../../../rpc/types';
+import { JSONRPCRequest } from '@matrixai/rpc/dist/types';
```

Comment:
`import type { JSONRPCRequest } from '@matrixai/rpc';`.

This is only possible if `@matrixai/rpc` does this:

```
export * from './types';
```

This is different from:

```
export * as types from './types';
```